addons/im_livechat/static/src/models/thread/thread.js

```diff
@@ -92,6 +97,25 @@ registerInstancePatchModel('mail.thread', 'im_livechat/static/src/models/thread/
     _computeIsChatChannel() {
         return this.channel_type === 'livechat' || this._super();
     },
+    /**
+     * @private
+     * @returns {boolean}
+     */
+    _computeisLiveChatActive() {
+        return (this.channel_type === 'livechat' && this.livechatActive !== undefined) ? this.livechatActive : true;
+    }
 });
 
+registerFieldPatchModel('mail.thread', 'im_livechat/static/src/models/thread/thread.js', {
+    livechatActive: attr({
+        'default': true
+    }),
+    isLiveChatActive: attr({
+        compute: '_computeisLiveChatActive',
+        dependencies: [
+            'channel_type',
+            'livechatActive',
+        ],
+    }),
```

Comment:
Just keep one value, which I would name `isLivechatActive`, but it should not have a default value (value must come from the server and stay undefined if it is not known or not applicable).